packages/core-components/src/lib/atoms/inputs/button-group/ButtonGroup.svelte

```diff
@@ -0,0 +1,55 @@
+<script>
+	import { setButtonGroupContext } from './lib';
+	import { derived, writable } from 'svelte/store';
+	import { INPUTS_CONTEXT_KEY } from '@evidence-dev/component-utilities/globalContexts';
+	import { getContext } from 'svelte';
+	import { buildInputQuery } from '@evidence-dev/component-utilities/buildQuery';
+	import ButtonGroupItem from './ButtonGroupItem.svelte';
+	import { page } from '$app/stores';
+	/** @type {string} */
+	export let name;
+	/** @type {string} */
+	export let title;
+
+	const inputs = getContext(INPUTS_CONTEXT_KEY);
+
+	let currentValue = null;
+	const valueStore = writable(null);
+	$: valueStore.update(() => currentValue);
+	$: $inputs[name] = currentValue?.value ?? null;
+
+	setButtonGroupContext(
+		(v) => (currentValue = v),
+		derived([valueStore], ([$v]) => $v)
```

Comment:
What's the point of the identity derived store vs. just `valueStore`?

Reply:
Makes the valueStore read-only in children

Reply:
https://svelte.dev/docs/svelte-store#readonly

Should switch to this after we're on >3.56.0